x-pack/plugins/transform/public/app/sections/transform_management/components/edit_transform_flyout/edit_transform_flyout_form.tsx

```diff
@@ -33,36 +33,130 @@ export const EditTransformFlyoutForm: FC<EditTransformFlyoutFormProps> = ({
-        dataTestSubj="transformEditFlyoutDocsPerSecondInput"
-        errorMessages={formFields.docsPerSecond.errorMessages}
-        helpText={i18n.translate(
-          'xpack.transform.transformList.editFlyoutFormDocsPerSecondHelptext',
-          {
-            defaultMessage:
-              'To enable throttling, set a limit of documents per second of input documents.',
-          }
-        )}
-        label={i18n.translate('xpack.transform.transformList.editFlyoutFormdocsPerSecondLabel', {
-          defaultMessage: 'Documents per second',
-        })}
-        onChange={(value) => dispatch({ field: 'docsPerSecond', value })}
-        value={formFields.docsPerSecond.value}
-      />
+
       <EditTransformFlyoutFormTextInput
         dataTestSubj="transformEditFlyoutFrequencyInput"
         errorMessages={formFields.frequency.errorMessages}
-        helpText={i18n.translate('xpack.transform.transformList.editFlyoutFormFrequencyHelptext', {
+        helpText={i18n.translate('xpack.transform.transformList.editFlyoutFormFrequencyHelpText', {
           defaultMessage:
             'The interval between checks for changes in the source indices when the transform is running continuously. Also determines the retry interval in the event of transient failures while the transform is searching or indexing. The minimum value is 1s and the maximum is 1h.',
         })}
         label={i18n.translate('xpack.transform.transformList.editFlyoutFormFrequencyLabel', {
           defaultMessage: 'Frequency',
         })}
         onChange={(value) => dispatch({ field: 'frequency', value })}
-        placeholder="1m"
+        placeholder={i18n.translate(
+          'xpack.transform.transformList.editFlyoutFormFrequencyPlaceholderText',
+          {
+            defaultMessage: 'Default: {defaultValue}',
+            values: { defaultValue: formFields.frequency.defaultValue },
+          }
+        )}
         value={formFields.frequency.value}
       />
+
+      <EuiSpacer size="l" />
+
+      <EuiAccordion
+        data-test-subj="transformEditAccordionDestination"
+        id="transformEditAccordionDestination"
+        buttonContent={i18n.translate(
+          'xpack.transform.transformList.editFlyoutFormDestinationButtonContent',
+          {
+            defaultMessage: 'Destination configuration',
+          }
+        )}
+        paddingSize="s"
+      >
+        <EditTransformFlyoutFormTextInput
+          dataTestSubj="transformEditFlyoutDestinationIndexInput"
+          errorMessages={formFields.destinationIndex.errorMessages}
+          label={i18n.translate(
+            'xpack.transform.transformList.editFlyoutFormDestinationIndexLabel',
+            {
+              defaultMessage: 'Destination index',
+            }
+          )}
+          onChange={(value) => dispatch({ field: 'destinationIndex', value })}
+          value={formFields.destinationIndex.value}
+        />
+
+        <EditTransformFlyoutFormTextInput
```

Comment:
There doesn't seem to be a way of removing the optional `pipeline` field having previously set it to a value. Here I attempted to save with an empty string:

![image](https://user-images.githubusercontent.com/7405507/93220909-06704500-f765-11ea-813d-c720efa294f6.png)

Reply:
Improved form validation and creating the update request object in 6410f323462.

Form validation is now able to identify relationships/dependencies between form fields, for example, if the `pipeline` field is left empty to reset it, the update request needs to pass on the `dest` part containing only the `index` attribute.

I included some more documentation in the commit linked above.